src/message-dispatch.js

```diff
@@ -122,6 +123,14 @@ export default class MessageDispatch extends EventTarget {
           this.scene.systems["hubs-systems"].soundEffectsSystem.playSoundOneShot(SOUND_QUACK);
         }
         break;
+      case "cube": {
```

Comment:
This will not be a "supported" feature and will likely go away in the future but is a useful way of testing the new networking system.